packages/angular/src/migrations/update-15-7-0/install-required-packages.ts

```diff
@@ -0,0 +1,30 @@
+import type { Tree } from '@nrwl/devkit';
+import { addDependenciesToPackageJson, readJson } from '@nrwl/devkit';
+import { getInstalledAngularMajorVersion } from '../../generators/utils/version-utils';
+import { getPkgVersionForAngularMajorVersion } from '@nrwl/angular/src/utils/version-utils';
+
+export default async function (tree: Tree) {
+  const packagesToInstall = [
+    '@angular-devkit/core',
+    '@angular-devkit/schematics',
+    '@angular-devkit/build-angular',
```

Comment:
Workspaces should already have `@angular-devkit/build-angular`, the init generator adds it to the workspace.